sdk/storage/storage-queue/review/storage-queue.api.md

```diff
@@ -187,20 +194,24 @@ export const logger: import("@azure/logger").AzureLogger;
 
 // @public
 export interface Logging {
-    deleteProperty: boolean;
+    delete: boolean;
```

Comment:
This is a breaking change, we should rename it in the convenience layer of have a transform

Reply:
Done